news/2023/2023-10-10-monthly-beatmapping-contest-october-2023.md

```diff
@@ -34,6 +34,9 @@ To keep things fresh, every MBC has a *limitation* that requires mappers to comp
 
 Given osu!taiko mapping is pretty limited to begin with, this contest is opting for a more song-oriented challenge: **your beatmap's song must be 240 BPM or faster.**
 
+- If your song contains variable BPM, the most common BPM must be 240 or higher.
+- Using "Double BPM" mapping where the song's BPM is half the beatmap's intended BPM is allowed, as long as the latter is 240 or higher.
```

Comment:
```suggestion
- A beatmap with double the rhythm density of an ordinary map (e.g. 1/8 rhythm functions like usual 1/4 rhythm) is allowed, so long as the song's doubled BPM is 240 or higher.
```
maybe worth adding an example here too, since words suck at explaining this